src/QueryNode/QueryForm.jsx

```diff
@@ -50,6 +64,29 @@ class QueryForm extends React.Component {
         <Select className='query-form__select' name='node_type' options={options} value={state.selectValue} onChange={this.updateValue} />
         <input className='query-form__input' placeholder='submitter_id' type='text' name='submitter_id' />
         <input className='query-form__search-button' type='submit' onSubmit={this.handleQuerySubmit} value='search' />
+        {
+          this.state.selectValue ? (
```

Comment:
Maybe a bit more to ask but is it easy to add some logic here to let the dropdown button only shows when there are nodes in the query page?

Currently the `Download All` button will show even if there is no nodes under a `node_type`, which is a bit weird.